Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/KeyboardShortcutModifier.swift

```diff
@@ -0,0 +1,156 @@
+    private var key: KeyEquivalent
+    /// The modifier keys that the user presses in conjunction with a key equivalent to activate the shortcut.
+    /// One of the `EventModifiers` enumerations.
+    ///
+    /// Possible values:
+    /// * `all`
+    /// * `caps_lock`
+    /// * `command`
+    /// * `control`
+    /// * `numeric_pad`
+    /// * `option`
+    /// * `shift`
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    private var eventModifiers: EventModifiers = []
+    
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        #if os(iOS) || os(macOS)
+        let value = try container.decode(String.self, forKey: .key)
+        
+        switch value {
+        case "up_arrow":
+            key = .upArrow
+        case "down_arrow":
+            key = .downArrow
+        case "left_arrow":
+            key = .leftArrow
+        case "right_arrow":
+            key = .rightArrow
+        case "clear":
+            key = .clear
+        case "delete":
+            key = .delete
+        case "end":
+            key = .end
+        case "escap":
+            key = .escape
+        case "home":
+            key = .home
+        case "page_up":
+            key = .pageUp
+        case "page_down":
+            key = .pageDown
+        case "return":
+            key = .return
+        case "space":
+            key = .space
+        case "tab":
+            key = .tab
+        case "a"..."z":
+            key = KeyEquivalent(value[value.startIndex])
+        case "A"..."Z":
+            key = KeyEquivalent(value[value.startIndex])
+        case "0"..."9":
+            key = KeyEquivalent(value[value.startIndex])
+        case "!","@","#","$","%","^","*","(",")","-","_","=","+","[","]","{","}","|",":",";","\"","'","<",">",",",".","/","?":
+            key = KeyEquivalent(value[value.startIndex])
+        default:
+            throw DecodingError.dataCorruptedError(forKey: .key, in: container, debugDescription: "invalid value for key")
+        }
+        
+        let modifiers = try container.decode([String].self, forKey: .eventModifier)
+        
+        for modifier in modifiers {
+            switch modifier {
+            case "all":
+                eventModifiers.insert(.all)
+            case "caps_lock":
+                eventModifiers.insert(.capsLock)
+            case "command":
+                eventModifiers.insert(.command)
+            case "control":
+                eventModifiers.insert(.control)
+            case "numeric_pad":
+                eventModifiers.insert(.numericPad)
+            case "option":
+                eventModifiers.insert(.option)
+            case "shift":
+                eventModifiers.insert(.shift)
+            default:
+                throw DecodingError.dataCorruptedError(forKey: .eventModifier, in: container, debugDescription: "invalid value for event_modifiers")
+            }
+        }
```

Comment:
This can be made an extension on `EventModifiers`. See [`Edge.Set`](https://github.com/liveview-native/liveview-client-swiftui/blob/d5028b8ea0a2fdaac13c97229d895824c29cb07d/Sources/LiveViewNative/Utils/Edge.swift#L46-L68) for an example of how to implement `Decodable` for an `OptionSet`.